targets/unity-v2/Testing/Editor/PlayFabPackager.cs

```diff
@@ -149,7 +151,7 @@ public static void MakeAndroidBuild()
         {
             Setup();
             SetScriptingBackend(ScriptingImplementation.Mono2x, BuildTarget.Android, BuildTargetGroup.Android); // Ideal setting for Android
-            SetIdentifier(BuildTargetGroup.Android, "com.PlayFab.PlayFabTest");
+			SetIdentifier(BuildTargetGroup.Android, TestPackageName);
```

Comment:
Please reset your editor preferences to use 4 spaces instead of tab-characters.
And then restore the tabbing in this file back to normal.